src/main/java/sonarquberepair/Main.java

```diff
@@ -1,66 +1,106 @@
 package sonarquberepair;
 
-import spoon.Launcher;
-import spoon.processing.Processor;
 import spoon.support.sniper.SniperJavaPrettyPrinter;
 
-import java.lang.reflect.Constructor;
+import com.martiansoftware.jsap.FlaggedOption;
+import com.martiansoftware.jsap.JSAP;
+import com.martiansoftware.jsap.JSAPException;
+import com.martiansoftware.jsap.JSAPResult;
+
 
 public class Main {
+	private final SonarQubeRepairConfig config = new SonarQubeRepairConfig();
+
+	public SonarQubeRepairConfig getConfig() {
+		return this.config;
+	}
+
+	public JSAP defineArgs() throws JSAPException{
+		JSAP jsap = new JSAP();
+
+		/* will be supporting multiple rules processing later so rulenumber(s) */
+		FlaggedOption opt = new FlaggedOption("ruleNumbers");
```

Comment:
`ruleNumbers` should be replaced by `ruleKeys` (check other places for similar situations and change them). We should keep out terminology as closest as possible to SonarSource.